javav2/usecases/creating_fsa_app/src/test/java/FSATests.java

```diff
@@ -0,0 +1,81 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+import com.example.fsa.FSAApplicationResources;
+import com.example.fsa.services.DetectSentimentService;
+import com.example.fsa.services.ExtractTextService;
+import com.example.fsa.services.PollyService;
+import com.example.fsa.services.S3Service;
+import com.example.fsa.services.TranslateService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+import java.io.IOException;
+import java.io.InputStream;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.lenient;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+public class FSATests {
+
+    @Mock
+    private ExtractTextService textService;
```

Comment:
Here, you're mocking the `ExtractTextService`. Later, you mock the implementation of `getCardText` to return 'Your mock text'. Then, you call `getCardText` _inside the test_ and verify that it was called. This test never actually touches your real service. Nothing is being tested here.

Reply:
This is modified now to test real code.